src/utils/ipfs/readIgnoreFiles.ts

```diff
@@ -1,17 +1,16 @@
-import fs from 'fs'
```

Comment:
Bikeshedding here, but this doesn't really have much to do with IPFS. Maybe `tasks/publish` is still the best place to leave it, and rename it to something like `createIgnorePatternFromFiles()`?